DTS-STN/Alpha-Site
# Text call to action component

# Description

[Create text with call to action component](https://trello.com/c/sAiv36Ew/187-187-create-text-with-call-to-action-component)

Create a text with call to action component so that we can dynamically reference them according to the data in the CMS.
![image](https://user-images.githubusercontent.com/72703030/117853803-d37c4380-b256-11eb-8680-8ea839e9f7bb.png)

## Acceptance Criteria

Should be able to pass in CSS classes in order to change sizing i.e. full width half width etc

## Test Instructions

1. Test the page in different screen sizes
2. Compare the component with the Figma mock-up
3. Check the code to see if there's anything to improve
4. Check if add different widths size in the className Prop work.

## Checklist


## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)
